runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingPlugin.java

```diff
@@ -107,6 +113,7 @@
 	private final List<TreeRespawn> respawns = new ArrayList<>();
 	private boolean recentlyLoggedIn;
 	private int currentPlane;
+	ClueNestTier clueTierSpawned;
```

Comment:
Reduce this visibility
```suggestion
	private ClueNestTier clueTierSpawned;
```